### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by tiagoalvesdulce**
As discussed on matrix, I added the "How to write the perfect PR" section to our contributing guidelines. Feedback is welcome.

### CODE REVIEW

**Comment by victorgcramos** [comment]
I think we should add a template for PRs, like the one below, because it improves the reviewing process. 

****
### Bug (or issue) description
> Quick explanation about the issue that the PR is aiming to solve

*Example*: This PR refers to issue [#issue_number](), that reports that some features are missing.

### Solution description
> Describe your code changes for reviewers. Explain your code changes and how does it solve the issue described

*Example:* This diff adds a new provider to handle the mentioned issue, so user can access it by using the new `useFeature()` hook, which implements the method A. Reference: [reference_link]()

#### Use Cases (if needed)
> If your PR adds a new feature, it might be a good idea to describe what is the proper use for it. Remember to clarify this in your code.

_Example:_ This hook can be used every time you want to fetch the User data.

### UI Changes Screenshot
> Add images that illustrate the UI changes, by adding at least 2 pics regarding the previous version and the one that the PR is implementing.

****

Also, I'd like to add a mention for the contributor to **go for a declarative approach**. I think [this guide](https://www.redotheweb.com/2015/09/18/declarative-imperative-js.html) is a good starting point for the ones that are not familiar with this approach.

**Comment by tiagoalvesdulce** [comment]
Pretty nice, @victorgcramos! I'll add the template to the contributing file before merging the PR.

**Comment by tiagoalvesdulce** [comment]
Requested the review of @fernandoabolafio and @thi4go. If you guys have anything to add, please LMK.

**Comment by victorgcramos** [comment]
@tiagoalvesdulce you can create a `PULL_REQUEST_TEMPLATE.md` file with the PR template so GitHub can apply it to new PRs

**Comment by tiagoalvesdulce** [comment]
That's really cool @victorgcramos, didn't know about that. Thanks for sharing!

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+> Describe your code changes for reviewers. Explain your code changes and how does it solve the issue described
+
+_Example:_ This diff adds a new provider to handle the mentioned issue, so user can access it by using the new `useFeature()` hook, which implements the method A. Reference: [reference_link]()
+
+#### Use Cases (if needed)
+
+> If your PR adds a new feature, it might be a good idea to describe what is the proper use for it. Remember to clarify this in your code.
+
+_Example:_ This hook can be used every time you want to fetch the User data.
+
```
Comment (victorgcramos) [inline PULL_REQUEST_TEMPLATE.md:18]:
```
### Dependencies
> Link the dependencies (if needed) from backend and/or pi-ui

*Example:* Depends on [politeia#pr]() and [pi-ui#pr]()
```

**Pull request merged**